crates/neon/src/sys/no_panic.rs

```diff
@@ -121,6 +121,12 @@ impl FailureBoundary {
     }
 }
 
+// HACK: Force `NAPI_PREAMBLE` to run without executing any JavaScript to tell if it's
```

Comment:
I'd add one more line to the comment that explains that `NAPI_PREAMBLE` is a macro in the implementation of Node-API that checks if it's legal to call into JS. Or at least link to something so someone can read more about what it is.